### PR TITLE
Update MyTime component to display today's date with day name and day number #1048

### LINKED ISSUE #1048: Rename today in MyTime
Replace current title for today with name of the day and the day number:
- Saturday 25th
- Monday 1st
- Friday 22nd
(th/st/nd in superscript)

This should only apply to today.

Why?
Because this is relevant information for the user and helps to distinguish today from the other days. The month and year are not shown - as that makes it too crowded and easier to remember than the day number - at least for me.

### PR DESCRIPTION
**Pull request by vinaybadgujar102**
Resolves #1048
This pull request updates the MyTime component to display today's date in the format "Saturday 25th", where "Saturday" is the long name of the day of the week, and "25th" is the day of the month with "th" in superscript. This formatting will only apply to today's date.

### CODE REVIEW

**Comment by tijlleenders** [comment]
Thanks a lot @vinaybadgujar102 !

Please do check before starting work like @shubhamku044 did - this will avoid some disappointments in the future.

@Tushar-4781 Can you merge this after check. Looks good to me.

File: src/pages/MyTimePage/MyTimePage.tsx [hunk 1]
```diff
@@ -55,6 +55,19 @@ export const MyTimePage = () => {
   const getDayComponent = (day: string) => {
     const colorIndex = -1;
     const freeHours = tasks[day]?.freeHrsOfDay;
+    
+    const today = new Date();
+    const dayOfMonth = today.getDate();
+    let suffix = "";
+    if (dayOfMonth === 1 || dayOfMonth === 21 || dayOfMonth === 31) {
```
Comment (Tushar-4781) [inline src/pages/MyTimePage/MyTimePage.tsx:62]: check whether the last character is equal to 1 or not. Avoid this approach of 3 checks

File: src/pages/MyTimePage/MyTimePage.tsx [hunk 2]
```diff
@@ -55,6 +55,19 @@ export const MyTimePage = () => {
...
+      suffix = "st";
+    } else if (dayOfMonth === 2 || dayOfMonth === 22) {
+      suffix = "nd";
+    } else if (dayOfMonth === 3 || dayOfMonth === 23) {
+      suffix = "rd";
+    } else {
+      suffix = "th";
```
Comment (Tushar-4781) [inline src/pages/MyTimePage/MyTimePage.tsx:69]: create a function in utils to get this suffix. Let's not crowd JSX file with these kind of functions

**Review by Tushar-4781 — CHANGES_REQUESTED** [review]

**Review by Tushar-4781 — CHANGES_REQUESTED** [review]
Please place a proper commit message.

**Comment by vinaybadgujar102** [comment]
Made the necessary changes as requested. Please review the changes.

File: src/pages/MyTimePage/MyTimePage.tsx (hunk 1 above)
Comment (vinaybadgujar102) [reply src/pages/MyTimePage/MyTimePage.tsx]: Solved it using alternate approach

File: src/pages/MyTimePage/MyTimePage.tsx (hunk 2 above)
Comment (vinaybadgujar102) [reply src/pages/MyTimePage/MyTimePage.tsx]: Created function in utils.js as requested

File: src/pages/MyTimePage/MyTimePage.tsx [hunk 3]
```diff
@@ -55,6 +55,10 @@ export const MyTimePage = () => {
   const getDayComponent = (day: string) => {
     const colorIndex = -1;
     const freeHours = tasks[day]?.freeHrsOfDay;
+    
+    const today = new Date();
```
Comment (Tushar-4781) [inline src/pages/MyTimePage/MyTimePage.tsx:59]: There's already a "today" variable declared in line number 23.

**Review by Tushar-4781 — CHANGES_REQUESTED** [review]

File: src/pages/MyTimePage/MyTimePage.tsx (hunk 3 above)
Comment (vinaybadgujar102) [reply src/pages/MyTimePage/MyTimePage.tsx]: I have made the necessary changes.

Sorry for the minute mistakes. I'm new to open source contributions and may make some mistakes, so any feedback or guidance would be greatly appreciated!

**Review by Tushar-4781 — APPROVED** [review]
LGTM

**Pull request merged**